py/mpconfig.h

```diff
@@ -757,6 +757,14 @@ typedef double mp_float_t;
 #define MICROPY_CAN_OVERRIDE_BUILTINS (0)
 #endif
 
+// Allow ports to provide a lazy load function for undefined globals
+// By enabling this, MicroPython will call mp_lazy_load_global() whenever an undefined
+// global is accessed.  Ports can return a non-MP_OBJ_NULL value which will be returned instead
```

Comment:
nitpick, no extra spaces at start of sentence before 'Ports' and 'It' below

Reply:
I'm no big fan of 2 spaces after a period either, but I noticed it's the general convention here so I'm just following it. (e.g see the comment just next on `MICROPY_BUILTIN_METHOD_CHECK_SELF_ARG`)

Reply:
Oops, never eben noticed that. Probably because it's also not applied consistently..

Reply:
Haha, it took me a while to notice, too. Been trying to follow that since. Can't say I've been strict about it.

Also, the Internet doesn't seem to like this habit: https://www.cultofpedagogy.com/two-spaces-after-period/